ops/services/app_functions/report_stream_batched_publisher/functions/QueueBatchedReportStreamUploader/lib.ts

```diff
@@ -0,0 +1,152 @@
+import { Context } from "@azure/functions";
+import {
+  DequeuedMessageItem,
+  QueueClient,
+  QueueServiceClient,
+  StorageSharedKeyCredential,
+} from "@azure/storage-queue";
+import * as csvStringify from "csv-stringify/lib/sync";
+import { ENV, uploaderVersion } from "./config";
+
+const {
+  REPORT_STREAM_BATCH_MINIMUM,
+  REPORT_STREAM_BATCH_MAXIMUM,
+  REPORT_STREAM_TOKEN,
+  REPORT_STREAM_URL,
+  AZ_STORAGE_ACCOUNT_KEY,
+  AZ_STORAGE_ACCOUNT_NAME,
+  AZ_STORAGE_QUEUE_SVC_URL,
+  TEST_EVENT_QUEUE_NAME,
+} = ENV;
```

Comment:
I really appreciate all these variables declared here, makes this very flexible and maintainable, in my opinion.

Reply:
Thanks!